index.js

```diff
@@ -0,0 +1,80 @@
+const addButton = document.querySelector('.button');
+const bookList = document.getElementsByClassName('book-list')[0];
+const inputTitle = document.querySelector('.title');
+const inputAuthor = document.querySelector('.author');
+const alertMessage = document.querySelector('#alert-message');
+let theBooks = [];
+
+// const storage = localStorage;
+
+function removeBook() {
+  theBooks = theBooks.filter((book) => +book.id !== +this.parentNode.childNodes[2].innerHTML);
+  this.parentNode.remove();
+
+  theBooks.forEach((book, i) => {
+    bookList.children[i].children[2].innerHTML = i;
+    book.id = i;
+  });
+  localStorage.setItem('booksArray', JSON.stringify(theBooks));
+}
+
+function creaateAndAppend(title, author, id) {
+  const bookContainer = document.createElement('div');
+  const titleBook = document.createElement('h5');
+  const authorBook = document.createElement('h5');
+  const idBook = document.createElement('h5');
+  const removeButton = document.createElement('button');
+  const breakLine = document.createElement('hr');
+
+  titleBook.innerHTML = title;
+  authorBook.innerHTML = author;
+  idBook.innerHTML = id;
+  removeButton.innerHTML = 'Remove';
+
+  // idBook.style.display = 'none';
```

Comment:
- [ ] [OPTIONAL] Pease remove unused or commented code before merging the project to the next step!